assets/sass/patterns/organisms/content-header.scss

```diff
@@ -25,6 +24,49 @@
 
 }
 
+// Content Header Grid Articles
+
+.content-header-grid {
+  padding: 0 24px;
+}
+
+.content-header-grid__main {
+  @include margin(24, "bottom");
+  @include padding(24, "bottom");
+  border-bottom: 1px solid $color-text-dividers;
+}
+
+.content-header-grid__side {
+  @include margin(24, "bottom");
+  @include padding(24, "bottom");
+  border-bottom: 1px solid $color-text-dividers;
+}
+
+@media only all and (min-width: 900px) {
```

Comment:
Also, wouldn't it be better to store the breakpoint value in a variable? e.g.

```
(min-width: $bkpt-size--medium)
```

Reply:
Medium is 730. wide is 900 but yes we can do that as it makes sense.